pkgs/servers/gotosocial/default.nix

```
@@ -41,22 +41,22 @@ buildGoModule rec {
   '';
 
   # tests are working only on x86_64-linux
-  doCheck = stdenv.isLinux && stdenv.isx86_64;
+  # doCheck = stdenv.isLinux && stdenv.isx86_64;
+  # checks are currently very unstable in our setup, so we should test manually for now
+  doCheck = false;
 
   checkFlags =
     let
       # flaky / broken tests
       skippedTests = [
         # See: https://github.com/superseriousbusiness/gotosocial/issues/2651
         "TestPage/minID,_maxID_and_limit_set"
-        # See: https://github.com/superseriousbusiness/gotosocial/pull/2760. Stop skipping
-        # this test when fixed for go 1.21.8 and above
-        "TestValidationTestSuite/TestValidateEmail"
       ];
     in
     [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
 
-  passthru.tests.gotosocial = nixosTests.gotosocial;
+  # test are unstable, see above
+  # passthru.tests.gotosocial = nixosTests.gotosocial;
```

Integration tests are fine actually
```suggestion
  passthru.tests.gotosocial = nixosTests.gotosocial;
```